symengine/utilities/matchpycpp/.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you remove this file? (You can also add this to `.gitignore`)

Reply:
I didn't notice I had this file committed.